spec/controllers/provider_foreman_controller_spec.rb

```diff
@@ -172,7 +172,7 @@
 
   context "#edit" do
     before do
-      set_user_privileges
+      stub_user(features: :all)
```

Comment:
Notice how nearly all of them were saying "I want a user who can see everything"